amoro-shade-guava-32/src/main/resources/META-INF/NOTICE

```diff
@@ -1,4 +1,4 @@
-amoro-shade-guava-32
```

Comment:
Why do we change this?

Reply:
This is an unexpected change, I have rollbacked it.